.eslintignore

```diff
@@ -0,0 +1 @@
+.jshintignore
```

Comment:
ah I see; I'm not sure how symbolic links work in git and cross platforms.. maybe we should use --ignore-path instead ?